DCA632FFFE4954B8.json

```diff
@@ -0,0 +1,188 @@
+      "enable": true,
+      "radio": 0,
+      "if": -400000
+    },
+    "chan_multiSF_4": {
+      "enable": true,
+      "radio": 0,
+      "if": -200000
+    },
+    "chan_multiSF_5": {
+      "enable": true,
+      "radio": 0,
+      "if": 0
+    },
+    "chan_multiSF_6": {
+      "enable": true,
+      "radio": 0,
+      "if": 200000
+    },
+    "chan_multiSF_7": {
+      "enable": true,
+      "radio": 0,
+      "if": 400000
+    },
+    "chan_Lora_std": {
+      "enable": true,
+      "radio": 1,
+      "if": -200000,
+      "bandwidth": 250000,
+      "spread_factor": 7
+    },
+    "chan_FSK": {
+      "enable": true,
+      "radio": 1,
+      "if": 300000,
+      "bandwidth": 125000,
+      "datarate": 50000
+    },
+    "tx_lut_0": {
+      "pa_gain": 0,
+      "mix_gain": 8,
+      "rf_power": -6,
+      "dig_gain": 0
+    },
+    "tx_lut_1": {
+      "pa_gain": 0,
+      "mix_gain": 10,
+      "rf_power": -3,
+      "dig_gain": 0
+    },
+    "tx_lut_2": {
+      "pa_gain": 0,
+      "mix_gain": 12,
+      "rf_power": 0,
+      "dig_gain": 0
+    },
+    "tx_lut_3": {
+      "pa_gain": 1,
+      "mix_gain": 8,
+      "rf_power": 3,
+      "dig_gain": 0
+    },
+    "tx_lut_4": {
+      "pa_gain": 1,
+      "mix_gain": 10,
+      "rf_power": 6,
+      "dig_gain": 0
+    },
+    "tx_lut_5": {
+      "pa_gain": 1,
+      "mix_gain": 12,
+      "rf_power": 10,
+      "dig_gain": 0
+    },
+    "tx_lut_6": {
+      "pa_gain": 1,
+      "mix_gain": 13,
+      "rf_power": 11,
+      "dig_gain": 0
+    },
+    "tx_lut_7": {
+      "pa_gain": 2,
+      "mix_gain": 9,
+      "rf_power": 12,
+      "dig_gain": 0
+    },
+    "tx_lut_8": {
+      "pa_gain": 1,
+      "mix_gain": 15,
+      "rf_power": 13,
+      "dig_gain": 0
+    },
+    "tx_lut_9": {
+      "pa_gain": 2,
+      "mix_gain": 10,
+      "rf_power": 14,
+      "dig_gain": 0
+    },
+    "tx_lut_10": {
+      "pa_gain": 2,
+      "mix_gain": 11,
+      "rf_power": 16,
+      "dig_gain": 0
+    },
+    "tx_lut_11": {
+      "pa_gain": 3,
+      "mix_gain": 9,
+      "rf_power": 20,
+      "dig_gain": 0
+    },
+    "tx_lut_12": {
+      "pa_gain": 3,
+      "mix_gain": 10,
+      "rf_power": 23,
+      "dig_gain": 0
+    },
+    "tx_lut_13": {
+      "pa_gain": 3,
+      "mix_gain": 11,
+      "rf_power": 25,
+      "dig_gain": 0
+    },
+    "tx_lut_14": {
+      "pa_gain": 3,
+      "mix_gain": 12,
+      "rf_power": 26,
+      "dig_gain": 0
+    },
+    "tx_lut_15": {
+      "pa_gain": 3,
+      "mix_gain": 14,
+      "rf_power": 27,
+      "dig_gain": 0
+    }
+  },
+  "gateway_conf": {
+    "gateway_ID": "DCA632FFFE4954B8",
+    "server_address": "eu1.cloud.thethings.network",
+    "serv_port_up": 1700,
+    "serv_port_down": 1700,
+    "servers": [
+      {
+        "gateway_ID": "DCA632FFFE4954B8",
+        "server_address": "eu1.cloud.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ]
```

Comment:
you are missing some additional metadata: lat/long, contact and description mainly, eg:
```
    ],
    "ref_latitude": 1.11111,
    "ref_longitude": 2.22222,
    "ref_altitude": 45,
    "contact_email": "youremail@gmail.com",
    "description": "my new lorawan gateway"
```